apps/test/applabTest.js

```diff
@@ -111,11 +112,142 @@ describe('applab: getIdDropdown filtering modes', function () {
   it('does not accidentally pick up superset classes', function () {
     // Make sure searching for elements with class ".chart" does not also pick
     // up elements with class ".chart-friend"
-    assert.deepEqual(AppLab.getIdDropdownFromDom_(documentRoot, '.chart'), [
+    assert.deepEqual(Applab.getIdDropdownFromDom_(documentRoot, '.chart'), [
       { "display": '"chart9"', "text": '"chart9"' }
     ]);
-    assert.deepEqual(AppLab.getIdDropdownFromDom_(documentRoot, '.chart-friend'), [
+    assert.deepEqual(Applab.getIdDropdownFromDom_(documentRoot, '.chart-friend'), [
       { "display": '"image1"', "text": '"image1"' }
     ]);
   });
 });
+
+describe('hasDataStoreAPIs', function () {
```

Comment:
I tend to either run everything, or just the subset of things I'm actively working on. Now that the full run takes ~3 minutes, things aren't too painful. Or you can run with --fast which is a little more than just applab, but cuts out turtle/maze and gets the run under a minute.